blockchain/stake/tickets.go

```diff
@@ -273,7 +274,8 @@ func InitDatabaseState(dbTx database.Tx, params *chaincfg.Params) (*Node, error)
 // checks to ensure that the database has not failed the upgrade process and
 // reports the current version.  Upgrades are also handled by this function,
 // when they become applicable.
-func LoadBestNode(dbTx database.Tx, height uint32, blockHash chainhash.Hash, header wire.BlockHeader, params *chaincfg.Params) (*Node, error) {
+func LoadBestNode(dbTx database.Tx, height uint32, blockHash chainhash.Hash,
```

Comment:
Don't split function definitions across lines please.  I know that a lot of the stake stuff unfortunately made that mistake, but we need to go in the other direction, not split them all up.